.github/workflows/root-ci-config/buildconfig/global.txt

```diff
@@ -100,12 +100,12 @@ tcmalloc=OFF
 test_distrdf_dask=ON
 test_distrdf_pyspark=ON
 testing=ON
-tmva-cpu=ON
+tmva-cpu=OFF
 tmva-gpu=OFF
-tmva-pymva=ON
+tmva-pymva=OFF
 tmva-rmva=OFF
 tmva-sofie=OFF
-tmva=ON
+tmva=OFF
```

Comment:
I agree with Lorenzo. However, `tmva-pymva` really has to be disabled: macos14, macbeta, alma9, and ubuntu22 all fail at configuration time because of it: there are some neural nets that get trained at configuration time, which fails with linker errors.

As Danilo said on Mattermost: only ATLAS offline processing (grid and online) use 6.28 and they will not suffer from that.

Reply:
This PR here reverts Danilos commit and switches of `tmva-pymva`: https://github.com/root-project/root/pull/15305